pkgs/tools/networking/netbird/default.nix

```diff
@@ -2,7 +2,7 @@
 , lib
 , nixosTests
 , nix-update-script
-, buildGoModule
+, buildGo123Module
```

Comment:
such overrides are preferred via all-packages.nix

https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults

Reply:
Ah, I see. Wasn't aware of that. I just quickly looked at a few Go packages in here to double check and saw that's how it was done. 
I'll keep that in mind for the future :)